### PR TITLE
remove references to PDF reading via PIL

### PR DESCRIPTION
**Pull request by garfieldnate**
As stated in the PIL documentation
[here](https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html?highlight=pdf#write-only-formats),
it can only write PDF's, not read them. Remove references to reading
PDF's via PIL from this page to avoid confusion.

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

### CODE REVIEW

**Comment by garfieldnate** [comment]
Not stale, just needs a quick review and a merge.

File: docs/source/model_doc/layoutlmv2.mdx [hunk 1]
```diff
@@ -157,7 +157,7 @@ from PIL import Image
 
 processor = LayoutLMv2Processor.from_pretrained("microsoft/layoutlmv2-base-uncased")
 
-image = Image.open("name_of_your_document - can be a png file, pdf, etc.").convert("RGB")
+image = Image.open("name_of_your_document - can be a png, jpg, etc.").convert("RGB")
```
Comment (NielsRogge) [inline docs/source/model_doc/layoutlmv2.mdx:160]:
```suggestion
image = Image.open("name_of_your_document - can be a png, jpg, etc. of your documents like PDFs.").convert("RGB")
```

File: docs/source/model_doc/layoutlmv2.mdx [hunk 2]
```diff
@@ -177,7 +177,7 @@ from PIL import Image
 
 processor = LayoutLMv2Processor.from_pretrained("microsoft/layoutlmv2-base-uncased", revision="no_ocr")
 
-image = Image.open("name_of_your_document - can be a png file, pdf, etc.").convert("RGB")
+image = Image.open("name_of_your_document - can be a png, jpg, etc.").convert("RGB")
```
Comment (NielsRogge) [inline docs/source/model_doc/layoutlmv2.mdx:180]:
```suggestion
image = Image.open("name_of_your_document - can be a png, jpg, etc. of your documents like PDFs.").convert("RGB")
```

File: docs/source/model_doc/layoutlmv2.mdx [hunk 3]
```diff
@@ -199,7 +199,7 @@ from PIL import Image
...
```
Comment (NielsRogge) [inline docs/source/model_doc/layoutlmv2.mdx:202]:
```suggestion
image = Image.open("name_of_your_document - can be a png, jpg, etc. of your documents like PDFs.").convert("RGB")
```

File: docs/source/model_doc/layoutlmv2.mdx [hunk 4]
```diff
@@ -219,7 +219,7 @@ from PIL import Image
 
 processor = LayoutLMv2Processor.from_pretrained("microsoft/layoutlmv2-base-uncased")
 
-image = Image.open("name_of_your_document - can be a png file, pdf, etc.").convert("RGB")
+image = Image.open("name_of_your_document - can be a png, jpg, etc.").convert("RGB")
```
Comment (NielsRogge) [inline docs/source/model_doc/layoutlmv2.mdx:222]:
```suggestion
image = Image.open("name_of_your_document - can be a png, jpg, etc. of your documents like PDFs.").convert("RGB")
```

File: docs/source/model_doc/layoutlmv2.mdx [hunk 5]
```diff
@@ -237,7 +237,7 @@ from PIL import Image
 
 processor = LayoutLMv2Processor.from_pretrained("microsoft/layoutlmv2-base-uncased", revision="no_ocr")
 
-image = Image.open("name_of_your_document - can be a png file, pdf, etc.").convert("RGB")
+image = Image.open("name_of_your_document - can be a png, jpg, etc.").convert("RGB")
```
Comment (NielsRogge) [inline docs/source/model_doc/layoutlmv2.mdx:240]:
```suggestion
image = Image.open("name_of_your_document - can be a png, jpg, etc. of your documents like PDFs.").convert("RGB")
```

**Review by NielsRogge — COMMENTED** [review]
Thanks for your PR, apologies if I caused confusion. You indeed need to first read your PDF as a JPG/PNG before PIL can read it.

**Comment by garfieldnate** [comment]
No problem, Niels. I think the wording you suggested is still a bit confusing, but since you want to mention PDF I'll send back a counter-edit :)

**Comment by garfieldnate** [comment]
@NielsRogge I have built on your comment, making it explicit that PDFs have to be converted to images first.

**Comment by garfieldnate** [comment]
Looks like I need to come back and style the file I changed.

**Comment by garfieldnate** [comment]
Fixed the line lengths.

**Comment by garfieldnate** [comment]
@NielsRogge I fixed the line lengths for the lines that we actually changed. The line length test is still failing, but it's because there are other lines in the file that are longer than 119 characters.

**Review by NielsRogge — APPROVED** [review]

**Review by LysandreJik — APPROVED** [review]

**Pull request merged**